datasource-streaming/README.md

```diff
@@ -8,7 +8,7 @@ the [Local Clients](../local-clients/README.md) example for its setup but uses a
 ## Prerequisites:
 
 **NOTE:** Streaming is only supported for S3 File Stream and GCS File Stream Connection types. (See [Connection Types](https://cognitivescale.github.io/cortex-fabric/docs/reference-guides/connection-types).)
-This example assumes an S3 File Stream is being used, but you can update Connection definition and Secrets accordingly.
+This example assumes an S3 File Stream is being used, but you can update Connection definition and Secrets accordingly. This example uses delta merge for batch ingestion, the merge requires a unique primary key in your dataset.
```

Comment:
Only suggestion might be bolding this sentence and/or adding a comment in the code about it